modules/theme/src/index.js

```diff
@@ -31,3 +31,11 @@ export const HomeTileIcons = {
     security: require("../lib/assets/images/home_security_mgt.png"),
     consent: require("../lib/assets/images/home_consent_mgt.png")
 };
+export const SidePanelIconSet = {
```

Comment:
Move icons to a folder

Reply:
Moved icons to a separate folder called `icons`